src/components/Search/index.tsx

```diff
@@ -107,8 +99,11 @@ const Search = ({ dispatch, hasBorder, content }: { dispatch: AppDispatch; hasBo
   const { searchBarEditable } = components
 
   useEffect(() => {
-    handleInputFocus(searchBarEditable, inputElement, dispatch)
-  }, [searchBarEditable, dispatch])
+    if (searchBarEditable) {
+      const input: HTMLInputElement = inputElement.current!
```

Comment:
It's better not to use the `!` assertion which makes the TypeScript meaningless.

Reply:
OK